src/app/modules/tests/services/search-edrs.service.ts

```diff
@@ -70,29 +71,21 @@ export class SearchEdrsService {
   }
 
 
-  searchEdrs(uri, params, auth?): Observable<any> {
-
-    let authorizationData: string = 'Basic ' + btoa(auth);
+  searchEdrs(uri, params, auth: {username: string, password: string}): Observable<any> {
+    let httpHeaders = new HttpHeaders().set('Content-Type', 'application/fhir+json');
     if (auth) {
-      const basicAuthString = `${auth.username}:${auth.password}`;
-      authorizationData = 'Basic ' + btoa(basicAuthString);
+      let basicAuthString = 'Basic ' + btoa(`${auth.username}:${auth.password}`);
```

Comment:
On line 77 you can replace let with const. This is very minor, but I try to use const anytime I declare a variable.